assets/javascripts/discourse/components/modal/discourse-post-event-invitees.js

```diff
@@ -0,0 +1,65 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { debounce } from "discourse-common/utils/decorators";
+
+export default class DiscoursePostEventInvitees extends Component {
```

Comment:
btw, we don't need the `Discourse` prefix in the name? could be a good opportunity to drop that 😃